kivy/__init__.py

```diff
@@ -251,6 +252,16 @@ def kivy_usage():
 #: Kivy user extensions directory
 kivy_userexts_dir = ''
 
+# if there are deps, import them so they can do their magic.
+try:
+    import kivy.deps
+    for importer, modname, ispkg in pkgutil.iter_modules(kivy.deps.__path__):
+        if not ispkg:
+            continue
+        importer.find_module(modname).load_module(modname)
+except ImportError:
+    pass
```

Comment:
At least, add a trace here. The day it wont work for any reason, you'll be glad to have a way for knowing where it fail ^^